packages/katello/rubygem-pulp_container_client/rubygem-pulp_container_client.spec

```diff
@@ -1,26 +1,25 @@
-# Generated from pulp_docker_client-4.0.0b6.dev01562331743.gem by gem2rpm -*- rpm-spec -*-
 # template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
-%global gem_name pulp_docker_client
+%global gem_name pulp_container_client
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 4.0.0b6.dev01562331743
+Version: 1.0.0rc1
```

Comment:
I think `rc1` here should be in `%prerelease`. Pretty sure this is making it fail in some way.